pyproject.toml

```diff
@@ -1,16 +1,39 @@
-[tool.poetry]
+[project]
 name = "adcm-aio-client"
-version = "0.1.0"
-description = "ADCM Client"
-authors = ["Aleksandr Alferov <aas@arenadata.io>"]
-license = "Apache License Version 2.0"
+description = "Asynchronous HTTP Client for ADCM (Arenadata Cluster Manager)"
+authors = [
+    { name = "Aleksandr Alferov", email = "<aas@arenadata.io>" },
+    { name = "Egor Araslanov", email = "<aer@arenadata.io>" },
+    { name = "Daniil Skrynnik", email = "<d.skrynnik@arenadata.io>" },
+    { name = "Artem Starovoitov", email = "<a.starovoitov@arenadata.io>" },
+    { name = "Vasiliy Chudasov", email = "<v.chudasov@arenadata.io>" }
+]
+license = "Apache-2.0"
+license-files = ["LICENSE"]
 readme = "README.md"
+requires-python = ">=3.12"
+keywords = ["ADCM"]
+dependencies = [
+    "httpx (>=0.27.2, <0.28.0)",
```

Comment:
it's quite strict restriction for a library

thou I see the versioning problem here, we don't use too many features of httpx, yet we probably don't want to track all new httpx versions for compatibility

Reply:
This entry `"httpx (>=0.27.2, <0.28.0)"` is equivalent to this `httpx = "^0.27.2"`.
What do you think if you restrict `"httpx (>=0.27.2, <1.0.0)"` ?